x-pack/docs/en/security/authentication/saml-guide.asciidoc

```diff
@@ -636,14 +636,14 @@ through either the
 NOTE: You cannot use {stack-ov}/mapping-roles.html#mapping-roles-file[role mapping files]
 to grant roles to users authenticating via SAML.
 
-This is an example of a simple role mapping that grants the `kibana_user` role
+This is an example of a simple role mapping that grants the `kibana_admin` role
 to any user who authenticates against the `saml1` realm:
 
 [source,console]
 --------------------------------------------------
 PUT /_security/role_mapping/saml-kibana
 {
-  "roles": [ "kibana_user" ],
+  "roles": [ "kibana_admin" ],
```

Comment:
I'd be wary of suggesting to add kibana_admin to all users of the realm as consumers of the docs tend to follow our examples verbatim